app/graphql/resolvers/freak_update_resolver.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class FreakUpdateResolver < Resolvers::Base
+    argument :id, GraphQL::Types::ID, required: true
+    argument :first_name, String, required: true
+    argument :last_name, String, required: true
+    argument :description, String, required: true
+    argument :email, String, required: true
+    argument :norm_id, GraphQL::Types::ID, required: true
+    argument :role_id, GraphQL::Types::ID, required: true
+    argument :level_id, GraphQL::Types::ID, required: true
+
+    type [Types::FreakType], null: false
+
+    # TODO: Test if norm_id exist
+    # TODO: Test if role_id exist
+    # TODO: Test if level_id exist
```

Comment:
You can put these on a single line

Reply:
Thanks for seeing that. I will change :)